### ISSUE TITLE
yarn run --> preview errors

### ISSUE THREAD

**Issue by mmattel** [owncloud/docs-ui#311]
Trying to update docs-ui and wanted to do a preview, I get errors...

`yarn run`
```
Which command would you like to run?: preview
$ gulp preview
[10:26:23] Using gulpfile /mnt/mm-klaus/www/owncloud/docs-ui/gulpfile.js
[10:26:23] Starting 'preview'...
[10:26:23] Starting 'build:preview'...
[10:26:23] Starting 'build'...
[10:26:25] gulp-imagemin: Minified 7 images (saved 23.1 kB - 91.9%)
[10:26:46] Finished 'build' after 23 s
[10:26:46] Starting '<anonymous>'...
[10:26:46] '<anonymous>' errored after 48 ms
[10:26:46] Error: Function yaml.safeLoad is removed in js-yaml 4. Use yaml.load instead, which is now safe by default.
    at Object.safeLoad (/mnt/mm-klaus/www/owncloud/docs-ui/node_modules/js-yaml/index.js:10:11)
    at /mnt/mm-klaus/www/owncloud/docs-ui/tasks/build-preview.js:49:29
    at async Promise.all (index 0)
    at async module.exports (/mnt/mm-klaus/www/owncloud/docs-ui/tasks/build-preview.js:13:30)
[10:26:46] 'build:preview' errored after 23 s
[10:26:46] 'preview' errored after 23 s
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Note, `yarn bundle` runs fine

Note, settingback: `"js-yaml": "^4.1.0",` --> `"js-yaml": "^3.14.1",` makes it work again. But this is not a nice solution...

@xoxys

**Comment by xoxys** [comment]
I'll look into a fix later this week.

**Comment by xoxys** [comment]
I'm wondering why it has stopped working now while all master branch builds were successful even with this `js-yaml` version...

**Comment by mmattel** [comment]
Dived deeper into the issue and found this:

The following files are also used in our docs-ui envorinment coming from the originator of antora. In particular the file `build-preview.js` needs an update.
 
https://git.cybbh.space/common/contributing-ui/-/tree/be69c4168cc57aa8091091e91f99879f5663ee22/tasks

There we see, that this has not been updated for a while reflecting the version changes and usages in js-yaml. I will file an issue in gitlab so this will get fixed there too.

When changing `build-preview.js` line 49:

`).then((contents) => yaml.safeLoad(contents));` --> 
`).then((contents) => yaml.load(contents));`

having js-yaml in a release > than 3.14 (as it is here using 4.1.x), the issue is solved.

> ...while all master branch builds were successful even with this js-yaml version

This may be the case because there was a successful build made in the past creating the necessary .zip file which is further used when running a docs build. But the update PR #312 might not create the necessary .zip file if this issue will not get fixed...

**Comment by mmattel** [comment]
Ok, more info:

We have to split the solution into two parts:
* Temporary fix as desribed above
* Long term fix accoring https://docs.antora.org/antora-ui-default/

**Comment by mmattel** [comment]
Closed via #313
